samples/nametranslation.d.ts

```diff
@@ -0,0 +1,5 @@
+declare const uint32Array: Uint32Array;
+
+declare const readonlyArray: ReadonlyArray;
+
+declare const promiseLike: PromiseLike;
```

Comment:
Please add an EOL at the end of the file so that git (and GitHub) don't freak out with that ugly symbol.